pkg/dynamicinstrumentation/testutil/sample/structs.go

```diff
@@ -5,6 +5,16 @@
 
 package sample
 
+type receiver struct{}
```

Comment:
Let's add a member field and an e2e test to check we are properly capturing the receiver, wdyt?

Reply:
I think positively about that idea!

Reply:
Ok i've added a member field and fixtures, but the pointer receiver on methods doesn't seem to be working properly (I think it used to?), i'll have to fix it but would rather do it in a follow up PR since i'm working on other stuff and would rather not have open PRs sitting around. I will prioritize fixing this before the next code freeze.